### ISSUE TITLE
Bad suggestion from import-name rule

### ISSUE THREAD

**Issue by chrismbarr** [Microsoft/tslint-microsoft-contrib#327]
I have this import statement
```ts
import UserSettings from "./user-settings.page";
```
which gives me this violation
```
3:1     import-name  Misnamed import. Import should be named 'user-settings.page' but found 'UserSettings'
```

but that suggestion is bad since I can't name a variable `user-settings.page` because it's not valid syntax.

Perhaps this should strip out invalid characters and attempt to camelCase them.

`user-settings.page` should give the suggestion `userSettingsPage`

**Comment by HamletDRC** [comment]
@chrismbarr I decided to go with userSettingsPage as a recommendation. There is nowhere else in the rule where we change the case of the suggestion, so it felt wrong to start capitalizing the first letter of the import in the case where the import name also contained a dash or a period.

**Comment by HamletDRC** [comment]
@chrismbarr the fix is on the #releases branch, and you can point your npm to that if you need to.
